### PR TITLE
Fe 1055 update pulse & d1 documentation

### PR DESCRIPTION
**Pull request by ipanagiv**
Update Box Content page for D1, Pulse

Update Installation instructions to include

External Battery Pack assembly & usage

Bird Spikes assembly

### CODE REVIEW

File: static/img/wxm-devices/pulse/pulse-contents-included-toremove.png
Comment (PavlosTze) [inline static/img/wxm-devices/pulse/pulse-contents-included-toremove.png:1]: The title says `toremove`, I guess this file should be deleted?

File: static/img/wxm-devices/d1/d1-contents-included-remove.svg
Comment (PavlosTze) [inline static/img/wxm-devices/d1/d1-contents-included-remove.svg:1]: The title says `remove`, I guess this file should be deleted?

File: docs/wxm-devices/pulse/assemble.mdx [hunk 1]
```diff
@@ -59,9 +59,9 @@ However, the wind vane’s movement has a small amount of friction, which assist
 
       Use a screwdriver to open the battery compartment and insert 2 AA batteries.
       Once the batteries are inserted, the LED indicator on the back of the sensor should turn on for 4 seconds
-      and then flash every 16 seconds indicating that the sensor transmits the data.
+      and then flash every 20 seconds indicating that the sensor transmits the data.
```
Comment (PavlosTze) [inline docs/wxm-devices/pulse/assemble.mdx:62]: Why is that change?

File: docs/wxm-devices/pulse/assemble.mdx [hunk 2]
```diff
@@ -59,9 +59,9 @@ However, the wind vane’s movement has a small amount of friction, which assist
...
       Note that the LED flashing only indicates the transmission of the data from the outdoor sensor
-      and it does not assure the reception of the data by the WG3000 Gateway.
+      and it does not assure the reception of the data by the WG1200 Gateway.
```
Comment (PavlosTze) [inline docs/wxm-devices/pulse/assemble.mdx:64]: Wrong, it's WG3000 in Pulse. Please be careful of copy-pasting :)

File: docs/wxm-devices/pulse/assemble.mdx [hunk 3]
```diff
@@ -79,6 +79,55 @@ thus no point in using rechargeable ones.
 :::danger
 Batteries should not be installed backwards. This may permanently damage the outdoor sensor.
 :::
+<hr/>
+### External Battery pack installation
+
+While not mandatory, we strongly recommend the installation of the battery pack for the following reasons:
+
+	1.	Extended Battery Life: Utilizing only two batteries will result in the station depleting its power within 3 to 5 months, depending on environmental conditions. Installing the battery pack increases the battery life by approximately four times, significantly reducing the frequency of battery replacements.
```
Comment (PavlosTze) [inline docs/wxm-devices/pulse/assemble.mdx:87]: utilising (British English)

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/wxm-devices/pulse/assemble.mdx (hunk 2 above)
Comment (ipanagiv) [reply docs/wxm-devices/pulse/assemble.mdx:64]: nice catch

**Review by PavlosTze — APPROVED** [review]

**Comment by gkoumanos** [comment]
- Arrows on batt pack are not 100% clear what they mean unless user reads the instructions
- Comment about lithium appears four times. Some texts are duplicated
- "Included Components" and "Installation Instructions" should have a style of a heading or something
- Rephrase "(consider referring to these as “spike rings” in the instructions)".

**Review by gkoumanos — APPROVED** [review]
LGTM

File: docs/wxm-devices/d1/assemble.mdx [hunk 4]
```diff
@@ -111,3 +178,5 @@ The straight edge of the metal plate is inserted in the groove at the bottom of
 The curved part of the 90-degree angle bent edge of the metal plate should be oriented to the wind sensor in order to “hug” the mounting pole.
 After inserting the metal plate, the U-bolts should be inserted through the respective holes of the metal plate.
 Loosely screw the nuts on the ends of the U-bolts. They will be tightened later when mounting on the pole (see [Mounting the outdoor sensor](/wxm-devices/m5/install-weather-station#mounting-the-outdoor-sensor)).
+
+
```
Comment (PavlosTze) [inline docs/wxm-devices/d1/assemble.mdx:182]: Remove these 2 empty lines

File: docs/wxm-devices/pulse/assemble.mdx [hunk 5]
```diff
@@ -110,4 +176,6 @@ The U-bolts and the metal plate are used to support the mounting of the outdoor
 The straight edge of the metal plate is inserted in the groove at the bottom of the unit (at the opposite side of the solar panel).
 The curved part of the 90-degree angle bent edge of the metal plate should be oriented to the wind sensor in order to “hug” the mounting pole.
 After inserting the metal plate, the U-bolts should be inserted through the respective holes of the metal plate.
-Loosely screw the nuts on the ends of the U-bolts. They will be tightened later when mounting on the pole (see [Mounting the outdoor sensor](/wxm-devices/pulse/install-weather-station#mounting-the-outdoor-sensor).
+Loosely screw the nuts on the ends of the U-bolts. They will be tightened later when mounting on the pole (see [Mounting the outdoor sensor](/wxm-devices/m5/install-weather-station#mounting-the-outdoor-sensor)).
+
+
```
Comment (PavlosTze) [inline docs/wxm-devices/pulse/assemble.mdx:181]: Remove these 2 empty lines.

File: docs/wxm-devices/d1/assemble.mdx [hunk 6]
```diff
@@ -81,6 +81,40 @@ Batteries should not be installed backwards. This may permanently damage the out
 :::
 <hr/>
 
+## External Battery pack installation
```
Comment (PavlosTze) [inline docs/wxm-devices/d1/assemble.mdx:84]: This needs to be `###`

File: static/img/wxm-devices/ws-1001/bird_spikes_1.png
Comment (PavlosTze) [inline static/img/wxm-devices/ws-1001/bird_spikes_1.png:1]: Please rename them and replace the underscore with a hyphen `-`, based on the naming conventions we use for the images.

File: static/img/wxm-devices/ws-1001/bird_spikes_2.png
Comment (PavlosTze) [inline static/img/wxm-devices/ws-1001/bird_spikes_2.png:1]: Please rename them and replace the underscore with a hyphen `-`, based on the naming conventions we use for the images.

File: static/img/wxm-devices/ws-1001/bird_spikes_3.png
Comment (PavlosTze) [inline static/img/wxm-devices/ws-1001/bird_spikes_3.png:1]: Please rename them and replace the underscore with a hyphen `-`, based on the naming conventions we use for the images.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

**Review by PavlosTze — APPROVED** [review]

**Pull request merged**